pages/blueprint/index.js

```diff
@@ -36,56 +36,8 @@ class BlueprintPage extends React.Component {
     this.handleChangeDescription = this.handleChangeDescription.bind(this);
 
     this.state = {
-      changes: [
-        {
-          commit: "3eaa3e0f732e37be4629042b8b74a4873ebb9909",
-          time: "Thu,  9 Nov 2017 14:50:41 +0000",
-          message: "These are comments about the changes that were committed."
-        },
-        {
-          commit: "627a776366f1f89e70d7453e1d7f4c88e9025229",
-          time: "Thu,  9 Nov 2017 14:48:49 +0000",
-          message:
-          "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur \
-          sagittis ullamcorper commodo. Pellentesque vitae arcu non eros \
-          tincidunt malesuada."
-        },
-        {
-          commit: "5fc425a221d207393ae3720eaea5cbb9308657c3",
-          time: "Wed, 18 Oct 2017 17:44:50 +0000",
-          message: "Etiam aliquet elit sit amet mauris pretium, ut hendrerit mauris lacinia."
-        },
-        {
-          commit: "8feb1e64d7e3f9d8a16ec0476bc76447c67d7f63",
-          time: "Wed, 18 Oct 2017 17:41:10 +0000",
-          message: "Nullam nisl tellus, finibus et porttitor quis, efficitur ac ante."
-        },
-        {
-          commit: "a41325a28174d53ad5e54d2868a0fd312875468f",
-          time: "Thu,  5 Oct 2017 19:54:42 +0000",
-          message:
-          "Mauris tincidunt, tellus id commodo fermentum, tellus nisi elementum \
-          nisi, vitae lacinia augue sem eget turpis."
-        },
-      ],
-      images: [
-        {
-          date_created: '2/06/17',
-          date_exported: '2/06/17',
-          user: 'Brian Johnson',
-          type: 'iso',
-          change: '3',
-          size: '2,345 KB',
-        },
-        {
-          date_created: '1/17/17',
-          date_exported: '1/17/17',
-          user: 'Brian Johnson',
-          type: 'iso',
-          change: '2',
-          size: '1,234 KB',
-        },
-      ],
```

Comment:
The UI associated with this data is hidden except for the [section header "Changes"](https://github.com/weldr/welder-web/pull/250/files#diff-a72936711756a1ad3bf1f32d6b45f83dR210). Should we just remove this chunk of UI, or conditionally display it when data is available?

![image](https://user-images.githubusercontent.com/21063328/41996801-031b9d20-7a24-11e8-81ce-37a3e1e83dae.png)